autoload/fzf_mru/actions.vim

```diff
@@ -21,6 +21,14 @@ function! fzf_mru#actions#options() abort
   return options
 endfunction
 
+function! s:p(...)
+  let preview_window = get(g:, 'fzf_preview_window', '')
+    if len(preview_window)
```

Comment:
Can we use `!empty` instead of `len`? I don't want it to work when `let g:fzf_preview_window = 0`